src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    private static final String LINE_BREAK = "______________________________________________\n";
+
+    public static void printList() { //prints list of tasks
+        System.out.println(LINE_BREAK);
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println( Integer.toString(i + 1) + "." + tasks.get(i).toString());
```

Comment:
Avoid complicated expressions. Perhaps you can consider abstrating the logic into multiple statements and methods to make the code more readable.